front/app/components/IdeaCards/IdeasWithFiltersSidebar/index.tsx

```diff
@@ -202,6 +264,19 @@ const IdeaCards = ({ ideaQueryParameters, onUpdateQuery }: Props) => {
     setFiltersModalOpened(false);
   }, []);
 
+  const [isCTABarVisible, setIsCTABarVisible] = useState(false);
+
+  useEffect(() => {
+    function checkCTABarVisibility() {
+      if (document.getElementById('project-cta-bar')) {
+        setIsCTABarVisible(true);
+      }
+    }
+
+    window.addEventListener('scrollend', checkCTABarVisibility);
+    return () => window.removeEventListener('scroll', checkCTABarVisibility);
+  }, []);
```

Comment:
I know this isn't ideal, but it is the only way I could reliable get whether there is a CTA bar in the DOM :thinking: Using useEffects and useState didn't always work, and I didn't want to use any refs and have to pass them down many components.

Adding the scrollend listener makes a few extra calls, but it makes sure we have the correct value here for isCTABarVisible.